frontend/src/v5/ui/components/viewer/cards/tickets/pinDetails/pinDetails.component.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { useState } from 'react';
+import CircledPlusIcon from '@assets/icons/outlined/add-circle-outlined.svg';
+import PinIcon from '@assets/icons/outlined/pin-outlined.svg';
+import CrossIcon from '@assets/icons/close.svg';
+import DeleteIcon from '@assets/icons/delete.svg';
+import { FormattedMessage } from 'react-intl';
+import { PinAction, PinActions, PinContainer, PinName, SettingLocationText } from './pinDetails.styles';
+
+export const PinDetails = () => {
+	const [isEditMode, setIsEditMode] = useState(false);
+	const [hasPin, setHasPin] = useState(false); // This can be removed when pin logic added
+
+	const onClickCancelSetPin = () => setIsEditMode(false);
+	const onClickDelete = () => setHasPin(false);
```

Comment:
I think this should also set the edit mode to false, in fact, if you add and then delete a pin, it will still be given as selected

Reply:
This stuff is temporary until the functionality for pins is added. I just did it in a way where all states could be achieved. You're right that deleting a pin should disable edit mode, but it'll probably get overwritten any way when the pin logic is added so I think it's best to leave it for now.